modules/Ecommerce/EcoProduct/Services/Dashboard/EcoProductDashboardCRUDService.php

```
@@ -132,4 +289,74 @@ public function getProductStatistics(): array
             ];
         }
     }
+
+    /**
+     * Get enhanced product statistics with new fields
+     */
+    public function getEnhancedProductStatistics(): array
+    {
+        try {
+            $companyId = tenant('id');
+            $products = EcoProduct::where('company_id', $companyId);
+
+            return [
+                'total_products' => $products->count(),
+                'active_products' => $products->where('is_visible', true)->count(),
+                'products_in_stock' => $products->whereNotNull('stock')->where('stock', '>', 0)->count(),
+                'low_stock_products' => $products->whereNotNull('stock')->where('stock', '<=', 10)->where('stock', '>', 0)->count(),
+                'out_of_stock_products' => $products->where('stock', 0)->count(),
+                'digital_products' => $products->where('type', 'digital')->count(),
+                'normal_products' => $products->where('type', 'normal')->count(),
+                'male_targeted' => $products->where('gender', 'male')->count(),
+                'female_targeted' => $products->where('gender', 'female')->count(),
+                'all_gender' => $products->where('gender', 'all')->count(),
+                'with_discounts' => $products->whereNotNull('discount_type')->whereNotNull('discount_value')->count(),
+                'shipping_included' => $products->where('shipping_included_in_price', true)->count(),
+            ];
+
+        } catch (\Exception $e) {
```

The fallback data is hardcoded which may mask real issues. Consider logging the exception and returning an empty array or null instead of fake data.